data_model/src/evaluate.rs

```diff
@@ -617,20 +616,16 @@ pub mod model {
     #[ffi_type(opaque)]
     pub enum EvaluationError {
         /// Failed due to math exception
-        #[display(fmt = "Math error")]
         Math(#[cfg_attr(feature = "std", source)] MathError),
-        /// Validation error
-        #[display(fmt = "Validation failed")]
+        /// Validation failed
         Validation(#[cfg_attr(feature = "std", source)] Box<ValidationFail>),
-        /// Value not found in context.
-        #[display(fmt = "{_0}: Value not found in context")]
+        /// `{0}`: Value not found in context
```

Comment:
```suggestion
        /// `{0}`: Value not found in the context
```

Sounds better, but I'm not sure